locales/ar-arb/messages.po

```diff
@@ -1630,16 +1629,16 @@ msgid "Unable to process your request"
 msgstr "غير قادر على معالجة طلبك"
 
 msgid "Unable to retrieve asset \"%s"
-msgstr "غير قادر على استرداد الأصل \"٪ s"
+msgstr "غير قادر على استرداد الأصل \"%s"
 
 msgid "Unable to retrieve item : "
 msgstr "تعذر استرداد التمرين:"
 
 msgid "Unable to retrieve item : \"%s"
-msgstr "تعذر استرداد البند: \"٪ s"
+msgstr "تعذر استرداد البند: \"%s"
 
 msgid "Unable to validate %s: %s"
-msgstr "غير قادر على التحقق من صحة٪ s:٪ s"
+msgstr "غير قادر على التحقق من صحة% s:% s"
```

Comment:
I don't think this is going to work with the space between the `%` and the `s`

Reply:
Fixed :)